docker/dev/docker-compose.yml

```diff
@@ -40,23 +44,62 @@ services:
         hard: -1
 
   postgres:
-    image: postgres
+    container_name: postgres
+    image: postgres:13.1-alpine
     environment:
       - POSTGRES_USER=timesketch
       - POSTGRES_PASSWORD=password
     restart: always
 
   redis:
-    image: redis
+    container_name: redis
+    image: redis:6.0.10-alpine
     restart: always
 
   notebook:
+    container_name: picatrix
```

Comment:
do we want to name the container picatrix? I would opt for just naming it `notebook`

Reply:
Good catch. Changed.